docs/source/tensorbay_cli/cli_commands.rst

```diff
@@ -1,3 +1,129 @@
 ##############
  CLI Commands
 ##############
+
+*************
+ gas dataset
+*************
+
+"gas dataset" deals with TensorBay datasets.
+
+Create a dataset.
+
+.. code:: console
+
+   gas dataset [dataset_name]
+
+Note that there is no tb: before the dataset name when creating a dataset.
+
+List all datasets.
+
+.. code:: console
+
+   gas dataset -l
+
+Delete a dataset.
+
+.. code:: console
+
+   gas dataset -d tb:[dataset_name]
+
+
+*********
+ gas ls
+*********
+
+"gas ls" deals with list operation.
+
+List the segments of a dataset.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]
+    gas ls tb:[dataset_name]@[revision]
+    gas ls tb:[dataset_name]#[draft_number]
+
+.. code:: console
+
+    gas ls tb:[dataset_name]:[segment_name]
+    gas ls tb:[dataset_name]@[revision]:[segment_name]
+    gas ls tb:[dataset_name]#[draft_number]:[segment_name
+
+Get the certain data.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]:[segment_name]://[remote_path]
+    gas ls tb:[dataset_name]@[revision]:[segment_name]://[remote_path]
+    gas ls tb:[dataset_name]#[draft_number]:[segment_name]://[remote_path]
+
+
+********
+ gas cp
+********
+
+"gas cp" deals with copy operation.
+
+Upload the all files in segments.
+
+.. code:: console
+
+    gas cp [local_path] tb:[dataset_name]#[draft_number]:[segment_name]
+
+Upload the files.
+
+.. code:: console
+
+    gas cp [local_path] tb:[dataset_name]#[draft_number]:[segment_name]://[remote_path]
+
+
+********
+ gas rm
+********
+
+"gas rm" deals with remove operation.
+
+Remove a segment.
+
+.. code:: console
+
+    gas rm -r tb:[dataset_name]#[draft_number]:[segment_name]
+
+Remove data.
+
+.. code:: console
+
+    gas rm tb:[dataset_name]@[revision]:[segment_name]://[remote_path]
+
+***********
+ gas draft
+***********
+
+"gas draft" deals with draft.
+
+Create a draft with a title.
+
+.. code:: console
+
+   gas draft tb:[dataset_name] -t [title]
+
+List the drafts.
+
+.. code:: console
+
+   gas draft -l tb:[dataset_name]
+
+
+***********
+ gas commit
+***********
+
+"gas commit" deals with commit.
```

Comment:
what this `commit` mean?
verb or noun？

Reply:
**gas commit**(boldface) deals with commit operation.